typings/styled-system.d.ts

```diff
@@ -0,0 +1,328 @@
+declare namespace StyledSystem {
```

Comment:
FYI Use of namespaces is discouraged, rather use `declare module` here directly and remove it altogether once it moves to DT.